### PR TITLE
Change the color pallet of navbar

### PR DESCRIPTION
**Pull request by hrushikesh-git**
## What is the change?
Improve the Color pallet of the navbar used the shades of Green from the following colour palette
color palate

And Changed the name from "Daily Journal" => "Quotidian Blogs" Quotadian is a synonym of Daily and bit of attractive.
navbar change

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -29,8 +29,8 @@ if(process.env.NODE_ENV !== 'production'){
 }
 
 //Connecting to Mongo Database using ODM Mongoose-
-const URL = process.env.URL;
-mongoose.connect(URL, {useNewUrlParser: true, useUnifiedTopology: true});
+const URL = 'mongodb://localhost:27017/blogDB';
+mongoose.connect(URL, {useNewUrlParser: true, useUnifiedTopology: true,useCreateIndex:true});
```
Comment (Pranav016) [inline app.js:33]: Revert back this change, local host is only for running the project on your local machine

**Review by Pranav016 — CHANGES_REQUESTED** [review]
* I think if the color of the whole navbar changes then that would be cool.. 
* Also I think a better color palate can be chosen, I like the color you chose for this, just a little brighter would do.
* Change base branch to develop
* I think the headline isn't very catchy/ attracting. `Quotidian BLOG'S`
I think a smaller word and either both words capitalized or only first letter looks subtle. Like `DAILY JOURNAL` or `Daily Journal`
* Resolve merge conflicts

**Comment by Pranav016** [comment]
/gssoc/

**Comment by hrushikesh-git** [comment]
@Pranav016  please review and merge the PR

File: views/partials/header.ejs [hunk 2]
```diff
@@ -40,22 +40,22 @@
...
+    <div class="container" id="nav">
       <div class="navbar-header">
         <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
           <span class="sr-only">Toggle navigation</span>
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>
         </button>
-        <a class="navbar-brand" href="/">DAILY <span class="navbar-brand-bold">JOURNAL</span></a>
+        <a class="navbar-brand" href="/"> <span class="navbar-brand-bold">Daily BLOG'S</span></a>
```
Comment (Pranav016) [inline views/partials/header.ejs:52]: * I want a new name, `Daily BLOG'S` doesn't seem good. Also I want the title to be either all caps or only first letter capital as I had requested earlier (you might've missed)
* Can you add a demo img of the new color used for the navbar

**Review by Pranav016 — CHANGES_REQUESTED** [review]

**Comment by hrushikesh-git** [comment]
navbar change

**Comment by Pranav016** [comment]
> * I think if the color of the whole navbar changes then that would be cool..
> * Also I think a better color palate can be chosen, I like the color you chose for this, just a little brighter would do.
> * Change base branch to develop
> * I think the headline isn't very catchy/ attractive `Quotidian BLOG'S`
>   I think a smaller word and either both words capitalized or only first letter looks subtle. Like `DAILY JOURNAL` or `Daily Journal`
> * Resolve merge conflicts

* Change base branch to develop as mentioned here. Take help from this on how to change 
image
* Name it `ALPHAVIO BLOGS`.
Otherwise looks good to merge.

**Comment by hrushikesh-git** [comment]
How to change base branch to develop ?

**Comment by RushanKhan1** [comment]
> > * I think if the color of the whole navbar changes then that would be cool..
> > * Also I think a better color palate can be chosen, I like the color you chose for this, just a little brighter would do.
> > * Change base branch to develop
> > * I think the headline isn't very catchy/ attractive `Quotidian BLOG'S`
> >   I think a smaller word and either both words capitalized or only first letter looks subtle. Like `DAILY JOURNAL` or `Daily Journal`
> > * Resolve merge conflicts
> 
> * Change base branch to develop as mentioned here. Take help from this on how to change
>   image
> * Name it `ALPHAVIO BLOGS`.
>   Otherwise looks good to merge.

@hrushikesh-git You can refer to the image attached above. You need to click on the button and then select ``develop``.

**Comment by hrushikesh-git** [comment]
Sir I have made the things correct please look into it and guide me

**Comment by AnanyaNegi** [comment]
> 
> 
> Sir I have made the things correct please look into it and guide me

Hi @hrushikesh-git .Can you attach the new screenshot of the navbar so I can have a look at the color?

**Comment by hrushikesh-git** [comment]
navbar change
The color e same as above but the title is Changed according to the instruction to  ALPHAVIO BLOGS. The title is changed

**Comment by hrushikesh-git** [comment]
> navbar change
> The color is same as above but the title is Changed according to the instruction to ALPHAVIO BLOGS. The title is changed

everytime I solve one conflict a new conflict arise and I this time conflict is inside the CSS where I have changed the navbar colors I am not understanding what to do now

**Comment by Pranav016** [comment]
> > navbar change
> > The color is same as above but the title is Changed according to the instruction to ALPHAVIO BLOGS. The title is changed
> 
> everytime I solve one conflict a new conflict arise and I this time conflict is inside the CSS where I have changed the navbar colors I am not understanding what to do now

Don't worry I'll fix it.

**Comment by RushanKhan1** [comment]
Hey @hrushikesh-git! Thanks for contributing! Actually, there have been some major changes to the code base to fix the bug that was in production also, there are still some more changes that need to be done therefore please **pull the updated code tomorrow** and make a new pr according to that else there will be a lot of merge conflicts.

**Pull request closed without merging**